ws-nextjs-app/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Simorgh - Next.JS",
   "scripts": {
-    "setupDevEnv": "cp ../envConfig/local.env .env && mkdir -p public && cp -r ../public/. public",
+    "setupDevEnv": "cp ../envConfig/local.env .env && sed 's/:7080/:7081/g' .env > .env.tmp && mv .env.tmp .env && mkdir -p public && cp -r ../public/. public",
```

Comment:
I know we're unlikely to change the port, but it is overridable via the `PORT` environment variable:

e.g. `PORT=7082 yarn dev` will run NextJS on 7082.

Do we need to use the value of the environment variable instead of hardcoding?

Reply:
Yeah that's fair ^^ . In this case, I think the intention of the script is just to standardise the `.env` for local setup so everything (e.g. Reverb) points consistently to 7081.

The `PORT` variable can still be overridden at runtime with yarn dev, so we shouldn’t need to handle it dynamically here

Reply:
Hmmm, it ignores my port override & just runs everything on 7081 😢

Reply:
This change here doesn't change the behaviour of the `process.env.PORT` though. That value, in the Next.js app at least, is set by the `-p` argument in dev-mode and the `PORT` env variable in `package.json` for production.

So any refernces in the actual app to `process.env.PORT` need to stay. That value isn't set by these `envConfig` files.

Reply:
If you try on `latest` and change the `dev` command to:
`"dev": "PORT=7085 yarn setupDevEnv && next dev -p 7081",`

The `-p` arg will take precedence.

Reply:
> Although do you imagine in the distant future, when all pages are off Express & on NextJS, would we use 7080 as the primary port, or will we just keep it 7081 4eva?

Not really sure tbh, maybe would move to `7080` since its a nicer number? 😅

Reply:
I think you've explained why setting `PORT` as an environment variable doesn't work - I thought it was due to `SIMORGH_BASE_URL`, but it's actually because the port number is hardcoded in the`dev` and `start` commands. So I think we're OK & apologies for being pedantic too!